corehq/apps/styleguide/examples/bootstrap5/htmx_pagination_table_view.py

```diff
@@ -0,0 +1,19 @@
+from corehq.apps.hqwebapp.tables.pagination import SelectablePaginatedTableView
+from corehq.apps.styleguide.examples.bootstrap5.htmx_pagination_data import generate_example_pagination_data
+from corehq.apps.styleguide.examples.bootstrap5.htmx_pagination_table import ExampleFakeDataTable
+
+
+class ExamplePaginatedTableView(SelectablePaginatedTableView):
+    """
+    This view returns a partial template of a table, along with its
+    page controls and page size selection. Its parent classes handle
+    pagination of a given queryset based on GET parameters in the request.
+
+    This view will be fetched by the "host" `HtmxPaginationView`
+    via an HTMX GET request.
+    """
+    urlname = "styleguide_b5_paginated_table_view"
+    table_class = ExampleFakeDataTable
+
+    def get_queryset(self):
+        return generate_example_pagination_data(100)
```

Comment:
I have not yet. I'm investigating that after I return from PTO in January. This is intentionally simple as it's just an example. It will likely involve a custom paginator class, I'm sure of it...especially with elasticsearch data.